common/wallet_tx.c

```diff
@@ -39,6 +48,59 @@ struct command_result *param_wtx(struct command *cmd,
         return NULL;
 }
 
+struct command_result *param_wtx_utxos(struct command *cmd,
+				 const char *name,
+				 const char *buffer,
+				 const jsmntok_t *tok,
+				 struct wallet_tx *wtx)
+{
+	size_t i;
+	const jsmntok_t *curr;
+	struct bitcoin_txid **txids;
+	unsigned int **outnums;
+	char **outpoint = tal_arr(tmpctx, char*, 0);
```

Comment:
nit: it's weird to me that you allocate this one inline with the declaration, but the others (`txids` and `outnums`) below.

Reply:
It seemed weird to me too : I was not aware of the convention @rustyrussell pointed out in his review (to not directly allocate a pointer I will free). Since I dont free this one (`tmpctx`) I thought it was ok to allocate it directly.
I have to admit I'm a little bit confused with which style to use each time I declare a variable.